08_book_titles/book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_writer :title
+
+  def title
+    capitalize_title(@title)
+  end
+
+  def capitalize_title(str)
+    str.capitalize!.split.map { |item| %w[and in the of a an].include?(item) ? item : item.capitalize }.join(' ')
```

Comment:
No need to use `capitalize!` with a bang, the `capitalize` one suffices.

Great job otherwise :100: